src/commands/external.rs

```diff
@@ -49,10 +50,19 @@ pub async fn execute_external_subcommand(
             }
         }
         Err(Error::NotFound(e)) => {
-            tracing::debug!("Tried to resolve {plugin_name} to plugin, got {e}");
-            eprintln!("Error: '{plugin_name}' is not a known Spin command. See spin --help.\n");
-            print_similar_commands(app, &plugin_name);
-            process::exit(2);
+            if plugin_name == "cloud" {
+                println!("The `cloud` plugin is required. Installing now.");
+                let plugin_installer = Install {
+                    name: Some("cloud".to_string()),
```

Comment:
To test this out now (before the cloud plugin is added to the plugins repository), replace the `name` field with  `local_manifest_src` that points to the local manifest after cloning the [`cloud-plugin` repo ](https://github.com/fermyon/cloud-plugin) 

```
local_manifest_src: Some("/Path/to/repo/cloud-plugin/plugin/cloud.json".into()),
```

Reply:
now added to the plugins repo so no need to modify